vendor/bat-native-ads/src/bat/ads/internal/logging_util.cc

```diff
@@ -12,6 +12,21 @@
 
 namespace ads {
 
+bool ShouldLogHeader(const std::string& header) {
+  const std::vector<std::string> excluded_headers = {
```

Comment:
Is it possible to use an "allow list" instead of a deny list here? It would be easier to do security reviews of new code that way.

Reply:
good suggestion, will adjust

Reply:
done